parser.go

```diff
@@ -148,3 +148,17 @@ func findTest(tests []*Test, name string) *Test {
 	}
 	return nil
 }
+
+func (r *Report) Failures() (count int) {
```

Comment:
Can you add a comment to this function? Also, the named parameter is not needed here, could you remove it as well as the naked return on line 163?